### PR TITLE
Añadiendo mis test cases

### PR DESCRIPTION
**Pull request by ErickaIsa**
Se añaieron mis test cases de la atarea del 2do Sprint

### CODE REVIEW

File: api_endpoints/mail_important_endpoints.py
Comment (AlisonGuzman2006) [inline api_endpoints/mail_important_endpoints.py:1]: The changes are reviewed and accepted

Comment (Diego-Claros17) [inline api_endpoints/mail_important_endpoints.py:1]: No changes

File: assertions/assertion_schemas.py
Comment (AlisonGuzman2006) [inline assertions/assertion_schemas.py:1]: The additions of code is accepted. Clean code.

Comment (Diego-Claros17) [inline assertions/assertion_schemas.py:1]: No problem

File: assertions/assertion_status.py
Comment (AlisonGuzman2006) [inline assertions/assertion_status.py:1]: The additions of code is accepted. Clean code.

File: assertions/schema_email_insert_image.py
Comment (AlisonGuzman2006) [inline assertions/schema_email_insert_image.py:1]: The additions of code is reviewed and accepted.

File: assertions/assertion_status.py
Comment (Diego-Claros17) [inline assertions/assertion_status.py:1]: Good but you could add  a message error like other similar methods

File: assertions/schema_email_insert_image.py
Comment (Diego-Claros17) [inline assertions/schema_email_insert_image.py:1]: GJ

File: payloads/mail_important/payload_email_insert_image.py
Comment (AlisonGuzman2006) [inline payloads/mail_important/payload_email_insert_image.py:1]: The additions of code are reviewed and accepted.

File: payloads/mail_important/payload_schema_email_insert_image.json
Comment (AlisonGuzman2006) [inline payloads/mail_important/payload_schema_email_insert_image.json:1]: The additons of code are reviewed and accepted

File: tests/mail_important/conftest.py
Comment (AlisonGuzman2006) [inline tests/mail_important/conftest.py:1]: The changes are reviewed and accepted

File: tests/mail_important/test_post_email_insert_image.py
Comment (AlisonGuzman2006) [inline tests/mail_important/test_post_email_insert_image.py:1]: The code additon is reviewed and accepted. Clean code.

**Review by AlisonGuzman2006 — APPROVED** [review]
Additions and changes in the code are reviewed and approved.

File: tests/mail_important/conftest.py
Comment (Diego-Claros17) [inline tests/mail_important/conftest.py:1]: your code is OK but pls add files in some of your validations in order to simulate accordly

File: tests/mail_important/test_post_email_insert_image.py
Comment (Diego-Claros17) [inline tests/mail_important/test_post_email_insert_image.py:1]: GJ

**Review by Diego-Claros17 — APPROVED** [review]
Thre is an opportunity to improve the code, but is ok. Well done Eri.

**Pull request merged**